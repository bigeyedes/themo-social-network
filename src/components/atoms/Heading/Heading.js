import styled from 'styled-components'

const Heading = styled.h1`
	color: grey;
	font-size: 1.5em;
	font-family: 'Sora', sans-serif;
	margin: 0;
`

export default Heading;