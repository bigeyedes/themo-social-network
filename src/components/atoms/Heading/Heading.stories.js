import React from 'react'
import { storiesOf } from '@storybook/react';
import Heading from './Heading';

storiesOf('Atoms/Headings', module)
	.add('Heading', () => <Heading>Heading</Heading>)
