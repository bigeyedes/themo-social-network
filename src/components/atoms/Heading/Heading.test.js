import React from 'react'
import renderer from 'react-test-renderer';
import 'jest-styled-components'
import Heading from './Heading'

describe("Render heading component", () => {
	test("Matches the snapshot", () => {
		const tree = renderer.create(<Heading />).toJSON()
		expect(tree).toMatchSnapshot() 
	});

	test("If have color grey", () => {
		const tree = renderer.create(<Heading />).toJSON();
		expect(tree).toHaveStyleRule('color', 'grey')
	});

	test("If have heading text", () => {
		const tree = renderer.create(<Heading />).toJSON();
		expect(tree).getByText('Heading')
	});

});