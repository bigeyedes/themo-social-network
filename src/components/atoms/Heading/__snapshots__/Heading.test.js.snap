// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render heading component Matches the snapshot 1`] = `
.c0 {
  color: grey;
  font-size: 1.5em;
  font-family: 'Sora',sans-serif;
  margin: 0;
}

<h1
  className="c0"
/>
`;
